README.md

```diff
@@ -36,6 +36,7 @@ module "app_alb" {
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
 | alb_certificate_arn | The ARN of the certificate to be attached to the ALB. | string | - | yes |
+| alb_ssl_policy | The SSL policy (aka security policy) for the container that specifies the TLS protocols and ciphers allowed. | string | `ELBSecurityPolicy-2016-08` | no |
```

Comment:
Should it be the SSL policy for the ALB, not the container?

Reply:
updated